README.md

```diff
@@ -31,18 +31,55 @@ create a `deno.json` file in the root of your project:
 write a typescript file `./deno/local-setup.ts` to specify the tools and versions you need. For
 example
 
-```
+``` typescript
 import {
   packages,
   forPlatform,
   getHostPlatform,
-  installTo
+  installTo,
+  mapPlatform,
+  MultiPlatform,
+  DownloadFile,
+  Installable,
+  withEnv,
+  setAlias,
+  binary
```

Comment:
My intention was for the example in the README to be simple and minimal. An example like this that demos custom packages would be better in a docs subdirectory I think.